src/sql-parser/src/parser.rs

```diff
@@ -2900,7 +2900,22 @@ impl<'a> Parser<'a> {
     fn parse_create_sink(&mut self) -> Result<Statement<Raw>, ParserError> {
         self.expect_keyword(SINK)?;
         let if_not_exists = self.parse_if_not_exists()?;
```

Comment:
It still looks like we allow `IF NOT EXISTS` in an `EXPLAIN SCHEMA` statement, but the docs seem to indicate otherwise.

Reply:
Left a comment here: https://github.com/MaterializeInc/materialize/pull/22972/files#r1385835768